clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -2488,7 +2518,10 @@ void TryCallOp::print(::mlir::OpAsmPrinter &state) {
   state << getExceptionInfo();
   state << ")";
   mlir::Value indirectCallee = isIndirect() ? getIndirectCall() : nullptr;
-  printCallCommon(*this, indirectCallee, getCalleeAttr(), state);
+  printCallCommon(*this, indirectCallee, getCalleeAttr(), state,
+    [this]() -> ::mlir::cir::ExtraFuncAttributesAttr {
+      return getExtraAttrs();
```

Comment:
As I mentioned in previous reviews, these lambdas are unnecessary, there are more simple and clean ways to go about this.